CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/502) fr
 - Updated test c# project to .NET 6.0
 - Fixed test fail on ubuntu-latest
 
+Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/508) from [Miguel Nieto](https://github.com/mnieto) the following:
+- Fix import-module vsteam loses previously stored configuration: [issue #493](https://github.com/MethodsAndPractices/vsteam/issues/493)
+- Fixed test Versions fail on local machine
+
```

Comment:
Can you please merge yours from main and update the changelog to the latest version 7.11.0?

Reply:
Do you mean 7.12.0?  7.11.0 is already released

Reply:
Yes 7.12. Sorry